### PR TITLE
Feat/ Added functionality to generator to include hash code and eqaulity method overrides for route argurment classes 

### PR DESCRIPTION
**Pull request by devRaphe**
This issue is a reference to [this issue assigned to me on github](https://github.com/Stacked-Org/stacked/issues/881)

### CODE REVIEW

**Review by FilledStacks — APPROVED** [review]
Awesome. This looks great!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 * add validate form to FormViewModel ([#3](https://github.com/Stacked-Org/generator/issues/3)) ([0d424cd](https://github.com/Stacked-Org/generator/commit/0d424cd646ec8d8aad7a57dbdc4ad5cda311556a))
 
+## 1.2.0
```
Comment (FilledStacks) [inline CHANGELOG.md:8]: This should not be under that latest version change. The latest should always be ontop

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Check the versioning please. Make sure you have the latest generator code in your branch and then bump the version from that.

File: CHANGELOG.md (hunk 1 above)
Comment (devRaphe) [reply CHANGELOG.md]: I have made the requested changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # [1.1.0](https://github.com/Stacked-Org/generator/compare/v1.0.0...v1.1.0) (2023-04-03)
 
+## 1.2.0
```
Comment (FilledStacks) [inline CHANGELOG.md:3]: This still doesn't make sense. We can't have v 1.2.0 under v 1.1.1. It should be the latest version

Comment (devRaphe) [reply CHANGELOG.md]: I thought differently, this has been resolved.

**Pull request merged**